docs/OV_Runtime_UG/supported_plugins/config_properties.md

```diff
@@ -1,240 +1,188 @@
+* Compiled model properties - Once a model has been compiled using the compile_model method, it has properties showing the model’s configuration parameters, such as the model’s priority, performance hint, and cache directory.
 
-OpenVINO runtime has two types of properties:
-- Read only properties which provide information about the devices (such as device name, thermal state, execution capabilities, etc.) and information about configuration values used to compile the model (`ov::CompiledModel`) .
-- Mutable properties which are primarily used to configure the `ov::Core::compile_model` process and affect final inference on a specific set of devices. Such properties can be set globally per device via `ov::Core::set_property` or locally for particular model in the `ov::Core::compile_model` and the `ov::Core::query_model` calls.
+Device and model properties are represented as a named variable with a given string name and a type. Each property has a key name and a value associated with it.
 
-An OpenVINO property is represented as a named constexpr variable with a given string name and a type. The following example represents a read-only property with a C++ name of `ov::available_devices`, a string name of `AVAILABLE_DEVICES` and a type of `std::vector<std::string>`:
-```
-static constexpr Property<std::vector<std::string>, PropertyMutability::RO> available_devices{"AVAILABLE_DEVICES"};
-```
+The following sections show how to query device and model properties and configure them. Refer to the [Hello Query Device C++](../../../samples/cpp/hello_query_device/README.md) and [Hello Query Device Python](../../../samples/python/hello_query_device/README.md) samples to see more example code showing how to get and set properties in user applications.
 
-Refer to the [Hello Query Device С++ Sample](../../../samples/cpp/hello_query_device/README.md) sources and the [Multi-Device execution](../multi_device.md) documentation for examples of using setting and getting properties in user applications.
+### Querying Available Devices
 
-### Get a Set of Available Devices
+To find all the available processing devices and accelerators in the system, use the `ov::Core::get_available_devices` (C++) or [Core.available_devices](api/ie_python_api/_autosummary/openvino.runtime.Core.html#openvino.runtime.Core.available_devices) (Python) method:
 
 Based on the `ov::available_devices` read-only property, OpenVINO Core collects information about currently available devices enabled by OpenVINO plugins and returns information, using the `ov::Core::get_available_devices` method:
 
-@sphinxtabset
-
-@sphinxtab{C++}
-
-@snippet docs/snippets/ov_properties_api.cpp get_available_devices
-
-@endsphinxtab
-
-@sphinxtab{Python}
+@sphinxdirective
 
-@snippet docs/snippets/ov_properties_api.py get_available_devices
+.. tab:: C++
+   .. code_block:: sh
+   
+   auto property_keys = core.get_property(“CPU”, ov::supported_properties);
```

Comment:
why code snippets are removed? We should not write code in the documentation directly.